yalla-coop/death
# cancel payment journey

Relates #452
testing:
1. rebuild DB
2. login using othman@shamla.com
3. go to profile and make subscription(should be premium user now)
4. click cancel subscription (then should see when your subscription ends)
5. to simulate the subscription reaches the end date
   1. install stripe cli https://stripe.com/docs/stripe-cli  =>  stripe listen --forward-to localhost:8080/api/hooks/stripe
   2. go to stripe dashboard https://dashboard.stripe.com/test/subscriptions
   3. find the canceled subscription
![image](https://user-images.githubusercontent.com/34629478/78122445-6f613c00-7415-11ea-891b-2912bf3837db.png)
   4. and from the three dots beside it select "Cancel subscription"
   5. user now can make new subscription
